src/rxToStream.ts

```diff
@@ -101,4 +89,40 @@ class ReadableObservableStream<T> extends stream.Readable {
             }
         }
     }
+
+    private subscribe() {
+        if (this._subscription) return;
```

Comment:
@vibl

The check is in the subscribe.

Reply:
Oops, sorry :-)

Reply:
I put it here to avoid double subscription.

The `if` could also be in the `_read()` function for a tiny-tiny bit more speed, but I thought it was better to have the
subscription logic in one place.

Reply:
Ok, although that makes the name of the method somewhat misleading (which is why I didn't think of looking into it).